dao/user.dao.js

```diff
@@ -29,7 +29,7 @@ function UserDao() {
       isAdmin: false,
       createdAt: new Date(),
       photo:
-        "https://images.unsplash.com/photo-1623584973952-182bcb43b8ad?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=480&q=80",
```

Comment:
Consider defining this as a constant so you can use a name that provides contextual meaning. You might even want to define this in `.env` or a config file to make it easy to change one image for another. For example, `photo: errorUserImage,`.